src/extensions/nativeExtensions/image/ImageActionRenderBtn.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <v-dialog
+    v-model="dialog"
+    :max-width="dialogMaxWidth"
```

Comment:
I have an idea how to make the code more flexible in this place so that the user can change any props of the `v-dialog`. Need to do in the likeness of [ `card-props`](https://github.com/iliyaZelenko/tiptap-vuetify#card-props) or `toolbar-attributes` using `v-bind`. @avgeeklucky do you agree?

Reply:
@avgeeklucky can we do this, or do you have no time?

Reply:
Sorry, I've been busy with a few things. Agreed, dialog-props would add more flexibility.

Reply:
Nice!